.cirrus.yml

```diff
@@ -681,11 +681,15 @@ podman_machine_task:
     # Required_pr_labels does not apply to non-PRs.
     # Do not run on tags, branches, [CI:BUILD], or [CI:DOCS].
     only_if: *not_tag_branch_build_docs
+    only_if: *is_pr
+    # FIXME: label driven tests are unpredictable in terms of triggering
+    # for machine tasks. Run them on all PRs for now.
+    #
     # This task costs about $4 per attempt to execute.
     # Only run it if a magic PR label is present.
     # DO NOT ADD THIS TASK AS DEPENDENCY FOR `success_task`
     # it will cause an infinate-block / never completing build.
-    required_pr_labels: test_podman_machine
+    #required_pr_labels: test_podman_machine
```

Comment:
I think you can just nuke this whole comment section now.

Reply:
done